### PR TITLE
Bumping minbar typescript to 2.0.

### PR DESCRIPTION
**Pull request by dzearing**

### CODE REVIEW

File: common/changes/master_2016-10-28-00-34.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "changes": [
+    {
+      "project": "@microsoft/gulp-core-build-typescript",
+      "comments": "Making TypeScript 2.0.3 the default."
```
Comment (cliffkoh) [inline common/changes/master_2016-10-28-00-34.json]: Does it not need a version?

File: common/temp_modules/npmx-web-library-build/package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "@microsoft/gulp-core-build": ">=0.11.0 <1.0.0",
     "@microsoft/gulp-core-build-karma": ">=0.4.0 <1.0.0",
     "@microsoft/gulp-core-build-sass": ">=0.5.0 <1.0.0",
-    "@microsoft/gulp-core-build-serve": ">=0.4.0 <1.0.0",
+    "@microsoft/gulp-core-build-serve": ">=0.5.4 <1.0.0",
```
Comment (cliffkoh) [inline common/temp_modules/npmx-web-library-build/package.json]: Does this need to be edited by hand?

Comment (dzearing) [inline common/temp_modules/npmx-web-library-build/package.json]: no... npmx generate did this.

File: common/changes/master_2016-10-28-00-34.json (hunk 1 above)
Comment (dzearing) [inline common/changes/master_2016-10-28-00-34.json]: nope. just a type.

File: common/temp_modules/npmx-gulp-core-build-typescript/package.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
     "through2": "~2.0.1",
     "tslint": "~3.15.1",
     "tslint-microsoft-contrib": "~2.0.10",
-    "typescript": "~1.8.10"
+    "typescript": "^2.0.3"
```
Comment (nickpape-msft) [inline common/temp_modules/npmx-gulp-core-build-typescript/package.json]: Did you mean to have a ^ here?

**Pull request merged**